docs/tutorial/tutorial.md

```diff
@@ -1029,7 +1029,7 @@ Add a method called `jumpTo` to the Game class:
   jumpTo(step) {
     this.setState({
       stepNumber: step,
-      xIsNext: (step % 2) ? false : true,
+      xIsNext: !(step % 2),
```

Comment:
I find having a lot of operators in a tight space can be hard to parse. I like it as a ternary.